properties/utils.py

```diff
@@ -43,6 +43,12 @@ def func_wrapper(self):
 
     return func_wrapper
 
+def filter_dict(cls, fields):
```

Comment:
Could do? I thought that `filter_dict` would be a bit clearer (that it is working on the serialized version). Not really that concerned one way or another about the name, though. Could also make it something like `filter_keys`.